Update to `Window.showDirectoryPicker()` and `Window.showSaveFilePicker()` and `Window.showOpenFilePicker()` (#29581)
mdn/content: files/en-us/web/api/window/showopenfilepicker/index.md

  - : An object containing options, which are as follows:

    - `excludeAcceptAllOption` {{Optional_Inline}}
      - : A boolean value that defaults to
        `false`. By default the picker should include an option to not apply
        any file type filters (instigated with the type option below). Setting this option
        to `true` means that option is _not_ available.
    - `id` {{Optional_Inline}}
      - : By specifying an ID, the browser can remember different directories for different
        IDs. If the same ID is used for another picker, the picker opens in the same
        directory.
    - `multiple` {{Optional_Inline}}
      - : A boolean value that defaults to `false`. When
        set to `true` multiple files may be selected.
    - `startIn` {{Optional_Inline}}
      - : A `FileSystemHandle` or a well known directory (`"desktop"`, `"documents"`,
        `"downloads"`, `"music"`, `"pictures"`, or `"videos"`) to open the dialog in.

>     - `types`

Author: ```suggestion
    - `types` {{Optional_Inline}}
```

      - : An {{jsxref('Array')}} of allowed file types to pick. Each
        item is an object with the following options:

        - `description` {{Optional_Inline}}

>           - : An optional description of the category of files types allowed. Default to be an empty string.

Reviewer: ```suggestion
          - : An optional description of the category of files types allowed. Defaults to an empty string.
```

### Exceptions

- `AbortError` {{domxref("DOMException")}}
  - : An AbortError is thrown if a user dismisses the prompt without making a selection or
    if a file selected is deemed too sensitive or dangerous to be exposed to the website.
- `SecurityError` {{domxref("DOMException")}}

>   - : An SecurityError is thrown if it was blocked by Same-Origin policy or it was not called via a user interaction such as a button press.

Reviewer: ```suggestion
  - : Thrown if the call was blocked by the [same-origin policy](/en-US/docs/Web/Security/Same-origin_policy) or it was not called via a user interaction such as a button press.
```
